daprdocs/content/en/developing-applications/building-blocks/pubsub/subscription-methods.md

```diff
@@ -192,6 +193,118 @@ func eventHandler(ctx context.Context, e *common.TopicEvent) (retry bool, err er
 
 The `/checkout` endpoint matches the `route` defined in the subscriptions and this is where Dapr sends all topic messages to.
 
+### Streaming subscriptions
+
+Streaming subscriptions are subscriptions defined in application code that can be dynamically stopped and started at runtime.
+Messages are pulled by the application from Dapr. This means no endpoint is needed to subscribe to a topic, and it's possible to subscribe without any app configured on the sidecar at all.
+Any number of pubsubs and topics can be subscribed to at once.
+As messages are sent to the given message handler code, there is no concept of routes or bulk subscriptions.
+
+> **Note:** Only a single pubsub/topic pair per application may be subscribed at a time.
+
+The example below shows the different ways to stream subscribe to a topic.
+
+{{< tabs Go>}}
+
+{{% codetab %}}
+
+```go
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/dapr/go-sdk/client"
+)
+
+func main() {
+	cl, err := client.NewClient()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	sub, err := cl.Subscribe(context.Background(), client.SubscriptionOptions{
+		PubsubName: "pubsub",
+		Topic:      "orders",
+	})
+	if err != nil {
+		panic(err)
+	}
+	// Close must always be called.
+	defer sub.Close()
+
+	for {
+		msg, err := sub.Receive()
+		if err != nil {
+			panic(err)
+		}
+
+		// Process the event
+
+		// We _MUST_ always signal the result of processing the message, else the
+		// message will not be considered as processed and will be redelivered or
+		// dead lettered.
+		// msg.Retry()
+		// msg.Drop()
+		if err := msg.Success(); err != nil {
+			panic(err)
+		}
+	}
+}
+```
+
+or
+
+```go
+package main
+
+import (
+	"context"
+	"log"
+
+	"github.com/dapr/go-sdk/client"
+	"github.com/dapr/go-sdk/service/common"
+)
+
+func main() {
+	cl, err := client.NewClient()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	stop, err := cl.SubscribeWithHandler(context.Background(),
+		client.SubscriptionOptions{
+			PubsubName: "pubsub",
+			Topic:      "orders",
+		},
+		eventHandler,
+	)
+	if err != nil {
+		panic(err)
+	}
+
+	// Stop must always be called.
+	defer stop()
+
+	<-make(chan struct{})
+}
+
+func eventHandler(e *common.TopicEvent) common.SubscriptionResponseStatus {
+	// Process message here
+    // common.SubscriptionResponseStatusRetry
+    // common.SubscriptionResponseStatusDrop
+			common.SubscriptionResponseStatusDrop, status)
+	}
+
+	return common.SubscriptionResponseStatusSuccess
+}
+```
+
+{{% /codetab %}}
+
+{{< /tabs >}}
+
```

Comment:
@hhunter-ms  Suggest adding here is this video at this point and at this timing from the community call today. 
"Watch this video for an overview of streaming subscription"

https://youtu.be/57l-QDwgI-Y?t=841